charts/telemetry-server/scripts/pre-init-checks.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -eu
+set -o pipefail
+
+echo "Checking PostgreSQL connectivity"
+
+check_table() {
+    db=$1
+    table=$2
+    result=$(PGPASSWORD="$TELEMETRY_USER_PASS" psql -h "$POSTGRES_HOST" -U "telemetry" -d "$db" -t -c "SELECT to_regclass('${table}');")
```

Comment:
Better for portability to explicitly use the env command when setting env vars for a command invocation...

```suggestion
    result=$(env PGPASSWORD="$TELEMETRY_USER_PASS" psql -h "$POSTGRES_HOST" -U "telemetry" -d "$db" -t -c "SELECT to_regclass('${table}');")
```

Reply:
Resolved in latest commit